frontend/src/components/account/inventory/Inventory.js

```diff
@@ -12,16 +12,149 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License. */
 import React, {Component} from 'react';
+import InventoryItems from "./InventoryItems";
+import { loading } from "../../utils/Utilities";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import { Link } from "react-router-dom";
+import "./Inventory.css";
+import ComponentWithHeader from "../../header/ComponentWithHeader";
 
-class Inventory extends Component {
+class Inventory extends ComponentWithHeader {
```

Comment:
If you're only expecting pantry items here, I'd call this Pantry in the UI.
If you're expecting everything here, call this "Staples"

'Inventory' is used for items you have in stock now, not for items you always have.